app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -1,5 +1,6 @@
 module CandidateInterface
   class ApplicationFormController < CandidateInterfaceController
+    before_action :redirect_to_additional_references_path, only: %i[show review]
```

Comment:
As discussed, we'll try to move this to the signin controller auth method so we don't need to do redirects here.

Reply:
I'm a bit confused about how to end-to-end test these without creating too much mess 
currently we are using this in the system tests: 
```
  def given_i_am_signed_in_as_a_candidate
    @candidate = create(:candidate)
    login_as(@candidate)
  end
```
and in this case, we never hit the `sign in controller`, maybe we could pair on this tomorrow if you have some time!